include/manifold/manifold.h

```diff
@@ -46,11 +46,59 @@ class CsgLeafNode;
  * @brief Mesh input/output suitable for pushing directly into graphics
  * libraries.
  *
- * This may not be manifold since the verts are duplicated along property
- * boundaries that do not match. The additional merge vectors store this missing
- * information, allowing the manifold to be reconstructed. MeshGL is an alias
- * for the standard single-precision version. Use MeshGL64 to output the full
- * double precision that Manifold uses internally.
+ * The core (non-optional) parts of MeshGL are the triVerts indices buffer and
+ * the vertProperties interleaved vertex buffer, which follow the conventions of
+ * OpenGL (and other graphic libraries') buffers and are therefore generally
+ * easy to map directly to other applications' data structures.
+ *
+ * The triVerts indices should form a manifold mesh: each of the 3 halfedges of
+ * each triangle should have exactly one paired halfedge in the list, defined as
+ * having the start index of one equal to the end index of the other and
+ * vice-versa. However, this is not always possible - consider e.g. a cube with
+ * normal vector properties. Shared vertices would turn the cube into a ball by
+ * interpolating normals - the common solution is to duplicate each corner
+ * vertex into 3, each with the same position, but different normals
+ * corresponding to each face. This is exactly what should be done in MeshGL,
+ * however we request two additional vectors in this case: mergeFromVert and
```

Comment:
Again, just write vertex `mergeFromVert[i]` is merged into vertex `mergeToVert[i]` or something.

Btw, if we duplicate `v` into `v1, v2, v3`, should we define the merges as `v1 -> v3, v1 -> v3` or `v1 -> v2, v1 -> v3` or `v1 -> v2, v2 -> v3`? Are they directed or just union (where direction doesn't matter)? We should say this to avoid ambiguities.